src/sql/parts/dashboard/dashboard.module.ts

```diff
@@ -51,7 +51,7 @@ import { JobHistoryComponent } from 'sql/parts/jobManagement/views/jobHistory.co
 let baseComponents = [DashboardHomeContainer, DashboardComponent, DashboardWidgetWrapper, DashboardWebviewContainer,
 					DashboardWidgetContainer, DashboardGridContainer, DashboardErrorContainer, DashboardNavSection, WebviewContent, WidgetContent,
 					ComponentHostDirective, BreadcrumbComponent, ControlHostContent, DashboardControlHostContainer,
-					JobsViewComponent, AgentViewComponent, JobHistoryComponent];
+					JobsViewComponent, AgentViewComponent, JobHistoryComponent, StepsViewComponent];
```

Comment:
please name this `JobStepsViewComponent`.  I'd like to keep `Job` prefix on these related classes.